src/Data/Array.purs

```diff
@@ -708,6 +1053,12 @@ foldRecM f a array = tailRecM2 go a 0
         pure (Loop { a: res', b: i + 1 })
 
 -- | Find the element of an array at the specified index.
+-- |
+-- | ```purescript
+-- | unsafePartial $ unsafeIndex ["a", "b", "c"] 1 = "b"
+-- | unsafeIndex ["a", "b", "c"] 10 -- runtime error
```

Comment:
If I remember correctly, this won't produce a runtime error. Instead, the result will be `undefined` - of course, this is likely to cause a runtime error when the result is used, but it could also just cause the program to behave incorrectly but continue running. Would you mind checking this and fixing the example if necessary?

Reply:
Also, you'll need an `unsafePartial` here too, presumably?